_content/gateways/faq/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: FAQ
+section: Running a gateway
+zindex: 200
+---
+
+# Gateway FAQ
+
+This page consolidates some of the most frequently asked questions on [The Things Gateway](../gateway/index.md).
+
+> This section contains the most frequent issues and queries pertaining to the The Things Gateway. If you find issues that are not covered in this page, feel free to reach out to [The Things Network community forum](https://www.thethingsnetwork.org/forum).
+
+#### Q. How do I get started with The Things Gateway?
+
+Getting started with The Things Gateway is a 3 step process:
+
+- First step - Connect the antenna and plug the gateway into a power outlet
+- Second step - Go to the website (https://ttn.fyi/activate)
+- Third step - Follow the instructions on the website
+
+
+#### Q. How can I reset my gateway?
+
+- Pressing the button shortly while powered on restarts the gateway.
+- Holding the button for 2 seconds while powered on it erased the WiFi settings.
+- Holding the button for 5 seconds while powered on it erases the WiFi settings and activation settings.
+- Holding the button while powering on it will erase the full serial flash
+
+#### Q. The second time I power my gateway, it takes time to start up. why?
+
+Upon every power cycle and every 24 hours, the gateway checks for new firmware. If the latest firmware is not downloaded yet it will download the new image and perform a reboot in order to install the image. This will take a few minutes. The process is indicated by a flashing power LED. Power cycling during this process is not dangerous, but this process will be restarted on the next start-up. So, please be patient
+
+#### Q. My gateway stays in AP mode. What do I do?
+
+This behaviour occurs when there is a glitch in the wifi connection of the gateway. When this happens, your gateway will turn into an AP(Access Point) and devices you used to set up the gateway can connect to it (your laptop or phone). To avoid this issue, you should ‘forget’ the wifi AP of the gateway on your phone/laptop.
```

Comment:
I think we can add a sentence here just to make more explicit what we mean by *forget*.